src/style.css

```
@@ -1 +1,64 @@
 /* Styles go here */
+body {
+  margin: 0%;
+}
+
+a:active {
+  color: orange;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background-color: white;
+}
+
+.logo_img {
+  display: block;
+  padding: 10px;
+  margin-left: 50px;
+  width: 40px;
+  height: 40px;
+}
```

```suggestion
.logo_img {
  display: block;
  width: 40px;
  height: 40px;
}
```
padding and margin here is redundant. margin better to set in header and in header exist property align-items: center.